core/sodasql/cli/cli.py

```diff
@@ -488,3 +498,143 @@ def scan(scan_yml_file: str, warehouse_yml_file: str, variables: tuple, time: st
+    if tool == "dbt":
+        try:
+            from dbt.contracts.results import TestStatus
+            from dbt.contracts.graph.compiled import CompiledSchemaTestNode
+            from sodasql import dbt as soda_dbt
+        except ImportError as e:
+            raise RuntimeError(
+                "Soda SQL dbt extension is not installed: $ pip install soda-sql-dbt"
+            ) from e
+        if dbt_manifest is None:
+            raise ValueError(f"Dbt manifest is required: {dbt_manifest}")
+        if dbt_run_results is None:
+            raise ValueError(f"Dbt run results is required: {dbt_run_results}")
+
+        warehouse_yml_parser = build_warehouse_yml_parser(warehouse_yml_file)
+        warehouse_yml = warehouse_yml_parser.warehouse_yml
+
+        soda_server_client = create_soda_server_client(warehouse_yml)
+        if not soda_server_client.api_key_id or not soda_server_client.api_key_secret:
+            raise ValueError("Missing Soda cloud api key id and/or secret.")
+
+        with dbt_manifest.open("r") as manifest:
+            model_nodes, test_nodes = soda_dbt.parse_manifest(json.load(manifest))
+        with dbt_run_results.open("r") as run_results:
+            parsed_run_results = soda_dbt.parse_run_results(json.load(run_results))
+
+        models_with_tests = soda_dbt.create_models_to_tests_mapping(
+            model_nodes, test_nodes, parsed_run_results
+        )
+
+        tests_in_run_results = [
+            (run_result, test_nodes[run_result.unique_id])
+            for run_result in parsed_run_results
+            if run_result.unique_id in test_nodes.keys()
+        ]
+
+        tests = [
+            Test(
+                id=Parser.create_test_id(
+                    test_expression=test_node.compiled_sql if isinstance(test_node, CompiledSchemaTestNode) else None,
+                    test_name=test_run_results.unique_id,
+                    test_index=index,
+                    context_column_name=test_node.column_name,
+                    context_sql_metric_name=None,
+                    context_sql_metric_index=None,
+                ),
+                title=Parser.create_test_title(
+                    test_expression=test_node.compiled_sql if isinstance(test_node, CompiledSchemaTestNode) else None,
+                    test_name=test_run_results.unique_id,
+                    test_index=index,
+                    context_column_name=test_node.column_name,
+                    context_sql_metric_name=None,
+                    context_sql_metric_index=None,
+                ),
+                expression=test_node.compiled_sql if isinstance(test_node, CompiledSchemaTestNode) else None,
+                metrics=None,
+                column=test_node.column_name,
+            )
+            for index, (test_run_results, test_node) in enumerate(tests_in_run_results)
+        ]
+
+        test_results = {
+            test_run_results.unique_id:
+            TestResult(
```

Comment:
As value we can add the number of failed rows, not _really_ a value, but makes most sense (for v1)

Reply:
Added this, but it does **not** show up in the Soda cloud.